docs/getting_started/features/frequency_based_services.md

```diff
@@ -3,6 +3,22 @@
 Frequency-based services can be used to model services that operate on a regular frequency, such as buses running every 10 minutes or subway services operating 2 minutes within specified time intervals.
 When modeling a Frequency-based service, `stop_times.txt` contains the relative times between stops in order to determine the times to be displayed to riders. 
 
+**Pre-requirements**: Implement Base features.
+
 | Files included                   | Fields included   |
 |----------------------------------|-------------------|
-|[frequencies.txt](/schedule/reference/#frequenciestxt)|`trip_id`, `start_time`, `end_time`, `headway_secs`, `exact_times` |
+|[frequencies.txt](/documentation/schedule/reference/#frequenciestxt)|`trip_id`, `start_time`, `end_time`, `headway_secs`, `exact_times` |
+
+??? note "Sample Data"
+    <p style="font-size:16px">
+    The following table shows two different trips operating at different frequencies, one with departures every 30 min not following a fixed schedule, and one with exact arrival and departure times at every stop every 15 minutes.
```

Comment:
It's hard to understand what you mean here. 

```suggestion
    The following sample shows two distinct trips: trip `AWE1` that runs every 30 min (`headway_secs=1800`), and trip `AWE2` that runs every 15min (`headway_secs=900`).
    
The 'exact times' field indicates whether the schedule follows the precise start time entered in the 'start_time' field.  
* trip `AWE1` departs every 30min from 6:10am to 12:00pm
* trip `AW2` departs at 6:00am, 6:15am, 6:30am, and so on.
```

Reply:
Text was updated, I wasn't able to properly render the formatting suggested so I had to make some adjustments but kept the same text.